lib/active_record_doctor/detectors/short_primary_key_type.rb

```diff
@@ -23,7 +23,7 @@ def detect
         tables(except: config(:ignore_tables)).each do |table|
           column = primary_key(table)
           next if column.nil?
-          next if bigint?(column) || uuid?(column)
+          next if bigint?(column) || uuid?(column) || !integer?(column)
```

Comment:
I think the problem is with auto-incremented integers, so maybe the check can actually be reduced to:

```ruby
next if !integer?(column) || bigint?(column)
```

What do you think? I think UUIDs don't have to be tackled explicitly.

Reply:
I think the `uuid?` method can now be removed, too.

Reply:
Ah, oops 💩 Removed.